runners/flink/flink_runner.gradle

```diff
@@ -178,6 +178,8 @@ dependencies {
   runtimeOnly project(":sdks:java:harness")
   if (flink_version.startsWith('1.10')) {
       compile "org.apache.flink:flink-connector-kafka-0.11_2.11:1.10-lyft20200508"
+      compile "com.amazonaws:aws-java-sdk-s3:1.10.6"
```

Comment:
Why do we need these?

Reply:
Apparently this [code](https://github.com/lyft/streamingplatform/blob/master/spflinkjava/streamingplatform-event-connector/src/main/java/com/lyft/streamingplatform/s3eventconnector/S3FileSystem.java) uses the Amazon S3 client API.